roles/base/tasks/main.yml

```diff
@@ -4,10 +4,20 @@
 - name: Install required system packages.
   apt: pkg={{ item }} update-cache=yes cache_valid_time=3600
   with_items:
+    # Base
     - build-essential
     - gcc
     - g++
+    # Sysadmin
+    - htop
+    - strace
+    - vim-nox
+    - mlocate
+    - curl
+    - molly-guard
+    # Security
```

Comment:
Some packages are installed by default anyway (curl, htop, vim) but it doesn't hurt.